app/models/population.rb

```diff
@@ -0,0 +1,56 @@
+class Population < ApplicationRecord
+  belongs_to :location
+
+  validates :date, presence: true
+  validates :operational_capacity, presence: true, numericality: { greater_than: 0 }
```

Comment:
Presumably a bunch of these have a sensible maximum, too?

Reply:
That might be refined while working on the further stories to edit the population data, but so far has only been specified as a mandatory positive integer.

Reply:
Okay